projectgenerator/qgs_project_generator.py

```diff
@@ -97,6 +103,11 @@ def initGui(self):
         self.iface.addPluginToDatabaseMenu(
             self.tr('Project Generator'), self.__about_action)
 
+        self.toolbar = self.iface.addToolBar('Project Generator')
```

Comment:
`self.tr('Project Generator')`

I think we also need to more lines to properly remember the visible state automatically and show it in the toolbar context menu.

```
self.toolbar.setObjectName("ProjectGeneratorToolbar")
self.toolbar.setToolTip(self.tr('Project Generator Toolbar'))
```